osquery/utils/tests/aws_util_tests.cpp

```diff
@@ -29,6 +29,22 @@ static const char* kAwsProfileFileEnvVar = "AWS_SHARED_CREDENTIALS_FILE";
 static const char* kAwsAccessKeyEnvVar = "AWS_ACCESS_KEY_ID";
 static const char* kAwsSecretKeyEnvVar = "AWS_SECRET_ACCESS_KEY";
 
+void SetEnvVar(const std::string& name, const std::string& value) {
```

Comment:
Is it possibly to use the abstractions we already have over set/unset env vars? https://github.com/facebook/osquery/blob/f9cb7149a9ac53c4aa563886d3bd37955876753f/osquery/core/process.h#L224